components/TimeLineChartCountyComparison.vue

```diff
@@ -41,9 +41,9 @@ export default {
       default: ''
     },
     chartData: {
-      type: Array,
+      type: Object,
```

Comment:
The chartData array always had a length of 0 and all the counties were added as named properties so having the type be object seems more appropriate, couldn't find another use of it being an array outside of a type declaration in one of the utils but there might be something I'm missing.

Reply:
I think it was an Array of Object before. It's okay as an Object as it has an key("alameda" etc) and value data collection.